cmd/algorand-indexer/daemon.go

```diff
@@ -199,6 +293,10 @@ func init() {
 
 	daemonCmd.Flags().StringVarP(&indexerDataDir, "data-dir", "i", "", "path to indexer data dir, or $INDEXER_DATA")
 
+	daemonCmd.Flags().StringVarP(&cpuProfile, "cpuprofile", "", "", "file to record cpu profile to")
+	daemonCmd.Flags().StringVarP(&pidFilePath, "pidfile", "", "", "file to write daemon's process id to")
+	daemonCmd.Flags().StringVarP(&configFile, "configfile", "c", "", "file path to configuration file (indexer.yml)")
+
 	viper.RegisterAlias("algod", "algod-data-dir")
```

Comment:
Here are the alias', I think something like `viper.RegisterAlias("data-dir", "indexer-data")` should do it.

Reply:
I think we actually have to do `viper.RegisterAlias("data-dir", "data")`